kubelet/tests/test_kubelet.py

```diff
@@ -876,20 +920,172 @@ def test_kubelet_stats_summary_not_available(monkeypatch, aggregator, tagger):
     check._retrieve_stats.assert_called_once()
 
 
-def test_system_container_metrics(monkeypatch, aggregator, tagger):
+def test_process_stats_summary_not_source(monkeypatch, aggregator, tagger):
     check = KubeletCheck('kubelet', {}, [{}])
-    monkeypatch.setattr(
-        check, '_retrieve_stats', mock.Mock(return_value=json.loads(mock_from_file('stats_summary.json')))
+    pod_list_utils = PodListUtils(json.loads(mock_from_file('pods_windows.json')))
+    stats = json.loads(mock_from_file('stats_summary_windows.json'))
```

Comment:
could you please test with both fixtures `stats_summary.json` and `pods_windows.json` to make sure we didn't break anything

Reply:
Added a test, though it should have been already there from previous change.